.github/workflows/playwright-pr.yml

```
@@ -0,0 +1,166 @@
+              comment += `- Total Tokens: ${stats.total}\n`;
+              comment += `- With Price: ${stats.withPrice}\n`;
+              comment += `- Missing Price: ${stats.withoutPrice}\n`;
+              comment += `- Coverage: ${coverage}%\n`;
+            }
+
+            // Add tokens without prices
+            const tokensWithoutPrices = Object.values(tokenInfo);
+            if (tokensWithoutPrices.length > 0) {
+              comment += '\n#### Tokens Missing Prices\n';
+              
+              // Sort tokens by number of browsers affected (descending)
+              tokensWithoutPrices.sort((a, b) => b.browsers.size - a.browsers.size);
+              
+              for (const token of tokensWithoutPrices) {
+                const browsersList = Array.from(token.browsers).join(', ');
+                comment += `- ${token.symbol} (${token.name})\n  - Missing in: ${browsersList}\n`;
+              }
+
+              comment += `\n**Total tokens missing prices: ${tokensWithoutPrices.length}**\n`;
+            } else {
+              comment += '\n✅ All tokens have prices\n';
+            }
+
+            // Post comment to PR
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: comment
+            });
+
+      - name: Upload HTML Report
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 7
+          if-no-files-found: ignore
+
+      - name: Stop Server
+        if: always()
+        run: |
+          pkill -f "yarn run serve" || true
+
+      # Set the job status based on analysis results, not test results
+      - name: Set Job Status
+        if: always()
+        run: |
+          # Exit with success regardless of test failures
```

This hides ALL the errors and we have no way of knowing if something went wrong with analysis preparation process.
In my opinion, we should make sure that the command that runs tests finishes with success status even if tests failed rather than overriding the whole job status.

We can achieve this by adding `|| true` or by putting `yarn` invocation in if condition, e.g.
```
if yarn playwright ...; then
 echo success
else
  echo failure
fi
```